workspaces/orchestrator/packages/backend/Dockerfile

```diff
@@ -40,6 +40,7 @@ WORKDIR /app
 # Copy files needed by Yarn
 COPY --chown=node:node .yarn ./.yarn
 COPY --chown=node:node .yarnrc.yml ./
+COPY --chown=node:node backstage.json ./
```

Comment:
SonarCloud is complaining here. I do not see any risk keeping it as it is. Or it should be fixed systematically for the rest of the Dockerfile as well.